### ISSUE TITLE
Packagist

### ISSUE THREAD

**Issue by ekandreas** [10up/ElasticPress#434]
Hi!
Please register the plugin at Packagist so that install with composer is possible!
https://packagist.org/packages/10up/

**Comment by ekandreas** [comment]
Sure, I could go via wpackagist-plugin/elasticpress but it's quite better without the wpcom dep.

**Comment by johnpbloch** [comment]
This is waiting on #435 and will be taken care of as soon as that gets merged in.

**Comment by dkotter** [comment]
#435 has now been merged into develop. Will be released as part of v1.9, which should happen within the next couple week.s

**Comment by johnpbloch** [comment]
In the meantime, Packagist already recognized the name change in develop: https://packagist.org/packages/10up/elasticpress

**Comment by ekandreas** [comment]
Great! Thanks!
